src/tokens/PositionItem.tsx

```diff
@@ -36,6 +41,13 @@ export const PositionItem = ({
       description: position.displayProps.description,
       balanceUsd: balanceUsd.toNumber(),
     })
+    earnPositionIds.includes(position.positionId)
+      ? navigate(Screens.EarnPoolInfoScreen, { pool: position as EarnPosition })
+      : position.displayProps.manageUrl
+        ? navigate(Screens.WebViewScreen, {
```

Comment:
@finnian0826 on iOS we might need to open these externally, but this should be confirmed with Product.

Reply:
I can check, the ticket specifically says to use an internal browser but I can check about that requirement.